src/app/modules/ix-feedback/feedback-dialog/feedback-dialog.component.html

```diff
@@ -91,13 +91,13 @@ <h1 matDialogTitle>{{ 'How would you rate this page?' | translate }}</h1>
       {{ 'EULA' | translate }}
     </button>
 
-    <ix-jira-oauth
+    <ix-oauth-button
       *ngIf="showJiraButton"
-      [formControl]="form.controls.token"
-      [label]="'Token' | translate"
-      [tooltip]="tooltips.token | translate"
-      [required]="true"
-    ></ix-jira-oauth>
+      testId="login-to-jira"
+      [oauthType]="oauthType.Jira"
+      [oauthUrl]="'https://support-proxy.ixsystems.com/oauth/initiate?origin='"
```

Comment:
Let's move it into class property.

Reply:
oauthUrl is used in other scenarios also

Reply:
I mean we should extract 'https://support-proxy' into a property of the feedback-dialog.component.